### PR TITLE
Make profile info visible in mobile

### PR DESCRIPTION
**Pull request by Ariadnigaki**
Fix for #3490 
[test](https://mf-geoadmin3.int.bgdi.ch/ltgar_profile_info_fix/index.html?lang=fr&mobile=false&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,KML%7C%7Chttps:%2F%2Fpublic.int.bgdi.ch%2FylXuj88mSQmvafwY2H_4uA&layers_visibility=false,false,false,false,true&layers_timestamp=18641231,,,,&X=261993.12&Y=681879.92&zoom=1&topic=ech)

### CODE REVIEW

**Comment by davidoesch** [comment]
Test Link?

**Comment by Ariadnigaki** [comment]
The Test Link is the following:
https://mf-geoadmin3.dev.bgdi.ch/ltgar/src/?lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,KML%7C%7Chttps:%2F%2Fpublic.dev.bgdi.ch%2FnXEBmCv3SSuN0LHUsKs3wA&layers_visibility=false,false,false,false,true&layers_timestamp=18641231,,,,&X=222875.00&Y=760116.15&zoom=1&topic=ech

**Comment by davidoesch** [comment]
Uhh the application does not load on my Nexus 6p

**Comment by oterral** [comment]
@Ariadnigaki  creates a test link using `make s3deploybranch` it will deploy your branch on integration.

**Comment by loicgasser** [comment]
jenkins test

**Comment by oterral** [comment]
@Ariadnigaki can you fix the profile's test please ? 

Error: http://jenkins.dev.bgdi.ch/job/PR_geoadmin3/158/console

Here: https://github.com/geoadmin/mf-geoadmin3/blob/master/test/specs/profile/ProfileService.spec.js#L126

File: src/components/profile/ProfileService.js [hunk 1]
```diff
@@ -415,144 +415,8 @@ goog.require('ga_urlutils_service');
...
-          g8.append('text')
-              .attr('class', 'ga-profile-icon-text')
-              .attr('font-size', '0.9em')
-              .attr('x', 585)
-              .attr('y', elevLabelY)
-              .style('text-anchor', 'start');
-
-          this.updateLabels();
-        };
+        this.updateLabels();
```
Comment (oterral) [inline src/components/profile/ProfileService.js]: bad indentation

File: src/components/profile/ProfileService.js [hunk 2]
```diff
@@ -625,7 +488,7 @@ goog.require('ga_urlutils_service');
           //for the sum of the slope/surface distances
           this.slopeDist = this.slopeDistance();
           //for the hiking time
-          this.hikTime = this.hikingTime();
+           this.hikTime = this.hikingTime();
```
Comment (oterral) [inline src/components/profile/ProfileService.js]: bad indentation

File: src/components/profile/partials/profile.html [hunk 3]
```diff
@@ -6,6 +7,51 @@
     {{coordinates[1] | measure:'distance':[' m']}}
   </div>
   <div class="ga-profile-tooltip-arrow"></div>
+  </div>
+  <div class="profile-icons">
+  <div class="wrapper">
+  <div class="ga-profile-elevation-difference" translate-attr="{title: 'profile_elevation_difference'}">
```
Comment (oterral) [inline src/components/profile/partials/profile.html]: bad indentations

File: src/components/profile/style/profile.less [hunk 4]
```diff
@@ -2,8 +2,8 @@
   position: relative;
   display: inline-block;
   width: 100%;
-  height: 145px;
-
+  height: 167px;
+
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Remove spaces

File: src/components/profile/partials/profile.html [hunk 5]
```diff
@@ -6,6 +7,51 @@
     {{coordinates[1] | measure:'distance':[' m']}}
   </div>
   <div class="ga-profile-tooltip-arrow"></div>
+  </div>
+  <div class="profile-icons">
```
Comment (oterral) [inline src/components/profile/partials/profile.html]: Use prefixed class name like `ga-profile-icons`

File: src/components/profile/partials/profile.html [hunk 6]
```diff
@@ -6,6 +7,51 @@
...
+  <div class="wrapper">
```
Comment (oterral) [inline src/components/profile/partials/profile.html]: Use prefixed class name like `ga-wrapper`

Comment (oterral) [inline src/components/profile/partials/profile.html]: Do we really need this wrapper ?

File: src/components/profile/style/profile.less [hunk 7]
```diff
@@ -104,4 +96,216 @@
       left: 65px;
     }
   }
+   
+   .profile-icons {
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Use `ga-profile-icons` and fix the indentation

File: src/components/profile/style/profile.less [hunk 8]
```diff
@@ -104,4 +96,216 @@
...
+      width: 100%;
+      height: 30px;
+     }
+   }
+
+   .wrapper {
+    position: relative;
+    display: inline-block;
+    width: 700px;
+}
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Bad indetation

File: src/components/profile/style/profile.less [hunk 9]
```diff
@@ -104,4 +96,216 @@
...
+
+   .wrapper div {
+    float: left;
+}
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Bad indetation

File: src/components/profile/style/profile.less [hunk 10]
```diff
@@ -104,4 +96,216 @@
...
+
+  .ga-profile-elevation-difference {
+   color: black;
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Bad indetation

File: src/components/profile/style/profile.less [hunk 11]
```diff
@@ -104,4 +96,216 @@
...
+      position: relative;
+      font-size: 0.9em;
+      width: 50px;
+      padding-left: 1px;
+     }
+
+     .fa {
+    
+     
+     }
```
Comment (oterral) [inline src/components/profile/style/profile.less]: useless clas definition

File: src/components/profile/style/profile.less [hunk 12]
```diff
@@ -104,4 +96,216 @@
...
+    
+     .fa-resize-vertical::before {
+      position: relative;
+      font-size: 0.9em;  
+     }
+}
+  
+  .ga-profile-elevation-up {
+   color: black;
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Bad indentation

File: src/components/profile/style/profile.less [hunk 13]
```diff
@@ -104,4 +96,216 @@
...
+  .ga-profile-elevation-up {
+   color: black; 
+  
+   
+   .ga-profile-icon-text {
+    position: relative;
+    font-size: 0.9em;
+    padding-left: 4px;
+    width: 50px;
+   }
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Bad indentation

File: src/components/profile/style/profile.less [hunk 14]
```diff
@@ -104,4 +96,216 @@
...
+   
+    .fa {
+    }
+
+    .fa-ga-elevation-up::before {
+     position: relative;
+     font-family: FontAwesome;
```
Comment (oterral) [inline src/components/profile/style/profile.less]: it seems there is a lot of duplicate css rules in this file can yout try to factorize a bit please.

File: src/components/profile/style/profile.less [hunk 15]
```diff
@@ -104,4 +96,216 @@
...
+  
+  .fa-clock-o::before {
+    position: relative;
+    font-family: FontAwesome;
+    font-size: 0.9em;
+    padding-left: 19px;
+    }
+   }
+
+ }
```
Comment (oterral) [inline src/components/profile/style/profile.less]: A lot of bad indentations in this file.

**Comment by oterral** [comment]
can you deploy a branch @Ariadnigaki  : `make s3deploybranch`

File: src/components/profile/style/profile.less [hunk 16]
```diff
@@ -104,4 +96,153 @@
       left: 65px;
     }
   }
-}
+   
+  .ga-profile-icons {
+    width: auto;
+    display: inline-block;
+    overflow-x: auto;
```
Comment (oterral) [inline src/components/profile/style/profile.less]: add also     `overflow-y: hidden;`

File: src/components/profile/style/profile.less [hunk 17]
```diff
@@ -104,4 +96,153 @@
...
+  }
+  
+  .ga-profile-elevation-up {
+   
+    .ga-profile-icon-text {
+      padding-left: 4px;
+    }
+    
+    .fa-ga-elevation-up::before {
+      font-family: FontAwesome;
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Remove this.

**Comment by Ariadnigaki** [comment]
@oterral thank you for your help and review! I made the additional changes and deployed the branch again

File: src/components/profile/style/profile.less [hunk 18]
```diff
@@ -104,4 +96,153 @@
...
+  }  
+
+  .ga-profile-elevation-down { 
+
+    .ga-profile-icon-text {
+      padding-left: 4px;
+    }
+    
+    .fa-ga-elevation-down::before {
+      font-family: FontAwesome;
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Remove this font-family rule everywhere.

**Comment by Ariadnigaki** [comment]
It's done

**Comment by oterral** [comment]
Thanks @Ariadnigaki it's perfect!!  @davidoesch do you valid the behavior?

**Comment by davidoesch** [comment]
Tested and OK. Thanks for the fix

**Comment by oterral** [comment]
congrats @Ariadnigaki !!!!

**Pull request merged**